integration/Jenkinsfiles/utest_aurora.groovy

```diff
@@ -0,0 +1,88 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     atimic
+ */
+
+properties([
+        [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60', numToKeepStr: '60', artifactNumToKeepStr: '1']],
+        disableConcurrentBuilds(),
+        [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/nuxeo/nuxeo/'],
+        [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+        [$class: 'ParametersDefinitionProperty', parameterDefinitions: [
+                // branch hardcoded for now
+                [$class: 'StringParameterDefinition', name: 'BRANCH', description: '', defaultValue: 'feature-NXP-22077-add-aurora'],
+                // can't work in a pipeline multi-branch with build PR because BRANCH_NAME contains PR-XXXXX
+                //[$class: 'StringParameterDefinition', name: 'BRANCH', description: '', defaultValue: BRANCH_NAME],
+                [$class: 'StringParameterDefinition', name: 'PARENT_BRANCH', description: '', defaultValue: 'master'],
+                [$class: 'StringParameterDefinition', name: 'DBPROFILE', description: '', defaultValue: 'pgsql'],
```

Comment:
looks like a constant, not a user input param

Reply:
Indeed, was a requirement because we use it somewhere, here I just wanted to test Nuxeo 
against Aurora so I took the one done in [NXBT-2322](https://jira.nuxeo.com/browse/NXBT-2322) and make it work while waiting for general availability of pipeline files.